### DISCUSSION TITLE
ordered more than i need

### DISCUSSION THREAD

**Discussion by grericht** [nliaudat/esp32_8ch_motor_shield#51] [Buy or Sell]
v58_rev_1.52 (screws)
u can have one for 12,50Eur fully assembeled without esp32
+ shipping in germany (2-5eur)
+ esp32 (5eur)

**Comment by KarstenBl** [comment]
Hello grericht,
it is a version you are currently using and it's working fine?
Then i would be interested, no case i assume?
BR
Karsten

Am Fr., 23. Feb. 2024 um 15:29 Uhr schrieb grericht <
***@***.***>:

> u can have one for 12Eur fully assembeled without esp32
>
>    - shipping in germany (2-5eur)
>    - esp32 (5eur)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nliaudat/esp32_8ch_motor_shield/discussions/51>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3VH3ZQW5GXOGBFTM4EBV6LYVCRURAVCNFSM6AAAAABDW2GYOOVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGI2TSMBQGM>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
--
Karsten Blume
***@***.***

**Reply by grericht** [reply]
Please excuse me, I was just before sending the order yesterday. I wanted to order 10 and give away 6 in the hope of getting it cheaper. Then shipping and customs were added and suddenly the price was well over 20 euros without ESP. That was too expensive for me at first and I'll see if I like the order even better. Actually, I would also like to change something in the design. I would like to have a permanently soldered ESP32 on the board. Maybe you can also get a 100x100 board, which would probably be cheaper?!

So I don't use it. But since I would have tried an ESPHome based version myself, I hope that it is much better than nothing ;)

How many boards would you take? A collective order will definitely be cheaper.

**Comment by KarstenBl** [comment]
Hi Grericht,
i need two boards.
Shipping to Germany - i assume you are a German guy, too.
You would order including AC or DC input parts?
BR
Karsten

Am Sa., 24. Feb. 2024 um 17:18 Uhr schrieb grericht <
***@***.***>:

> Please excuse me, I was just before sending the order yesterday. I wanted
> to order 10 and give away 6 in the hope of getting it cheaper. Then
> shipping and customs were added and suddenly the price was well over 20
> euros without ESP. That was too expensive for me at first and I'll see if I
> like the order even better. Actually, I would also like to change something
> in the design. I would like to have a permanently soldered ESP32 on the
> board. Maybe you can also get a 100x100 board, which would probably be
> cheaper?!
>
> So I don't use it. But since I would have tried an ESPHome based version
> myself, I hope that it is much better than nothing ;)
>
> How many boards would you take? A collective order will definitely be
> cheaper.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/nliaudat/esp32_8ch_motor_shield/discussions/51#discussioncomment-8577953>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A3VH3ZXRC2GZRZC6VD5YL53YVIHETAVCNFSM6AAAAABDW2GYOOVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DKNZXHE2TG>
> .
> You are receiving this because you commented.Message ID:
> <nliaudat/esp32_8ch_motor_shield/repo-discussions/51/comments/8577953@
> github.com>
>


-- 
--
Karsten Blume
***@***.***

**Comment by grericht** [comment]
now i ordered 5 pcs fully assembeled. 1 more than i need.
the lowest cost was 137usd per 5 pcs.
1 for 27eur + shipping (in germany)

**Reply by nliaudat** [reply]
The cost is huge ! What shipping have you selected ?

**Reply by grericht** [reply]
cheapest i found. i took pcbway dhl because thise was nearly same price like fedex on jlcpcb

pcbway:
![Screenshot 2024-03-05 075741](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/128326089/f6a0163f-5828-464f-8ad6-b0ce406cbcfe)

jlcpcb:
![Screenshot 2024-03-05 081634](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/128326089/97522bc2-b3db-4abd-aceb-14c93ad42f16)

shipping 01
![Screenshot 2024-03-05 081906](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/128326089/8da7b46c-eaed-44ba-bae0-94c0216a8b63)

shipping 02
![Screenshot 2024-03-05 081934](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/128326089/008cb379-9fb9-4669-90da-03900945cac7)

did i something wrong?

**Reply by grericht** [reply]
pcbway dont start production. i think i can stop and refund?!

they ask now:
"About your order W639497AS1Y2_Gerber_esp32-8ch-dc-motor-shield_AC_PCB-v58-rev-1.52_2024-02-20.zip,here is an engineering question for you.

As shown in the attached pic 1,we can not identify the board shape according to the GKO layer of your gerber files. So pls check the attached pic 2,may I ask we should refer to the outline indicated by the purple arrow or the outline indicated by the yellow arrow to produce the board shape?Here are two suggestions for you.
A:the outline indicated by the purple arrow
B:the outline indicated by the yellow arrow
Pls select A or B."

i saw pictures with a and b. what is better or right?
![2](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/128326089/1acbc9c3-b473-4101-bf73-bde495726529)

**Comment by nliaudat** [comment]
For shipping, you can select "Economical global line" at jlcpcb


![image](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/6782613/279a6347-5146-441d-8682-f4e617bb10e7)

**Reply by grericht** [reply]
Economical global line is only available up to 1,5kg. so i have to deactivate the power supply. costs then 107$
after deactivating the esp32 pins and the cable connections the cost where 80$
hope to find the parts. soldering will be no problem for me.
i hope i can cancel the pcbway order.
but with power supply and cable connections and the esp-holder-pins i will reach 100$
20$ per board

**Reply by nliaudat** [reply]
They have change the rules. Here is my last order
![image](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/6782613/951682e0-e803-44f0-8ea2-6cc5d30b69ef)

**Reply by nliaudat** [reply]
The problem is on weight estimation :

![image](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/6782613/6fb1651c-8a0e-4522-86ef-b27af17ba940)

and now 2 kg

**Reply by grericht** [reply]
but this was without power supply onboard?

**Reply by nliaudat** [reply]
A full board (old version) is less than 75g : 

![2024-03-07 18 41 47](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/6782613/ddbc10fa-db9b-45dc-9761-1236f59f5c2e)

**Comment by nliaudat** [comment]
The A shape cause there is a hole to not infer with antenna
![image](https://github.com/nliaudat/esp32_8ch_motor_shield/assets/6782613/439b971a-c2c4-467b-b1fd-72bff35e2b32)
